src/mpi/romio/adio/ad_pvfs2/ad_pvfs2_write_list_classic.c

```diff
@@ -235,8 +235,8 @@ void ADIOI_PVFS2_OldWriteStrided(ADIO_File fd, const void *buf, int count,
     /* already know that file is noncontiguous from above */
     /* noncontiguous in file */
 
-/* filetype already flattened in ADIO_Open */
-    flat_flie = ADIOI_Flatten_and_find(fd->filetype);
+    /* filetype already flattened in ADIO_Open */
```

Comment:
OK, yeah, a "flat_flie" is definietely bogus.  Thanks for catching that.  However, we still need to find a flattened representation.  Just fix the typo.